tests/app/submitter/test_convert_payload_0_0_3.py

```diff
@@ -1535,3 +1562,63 @@ def test_payload_dynamic_answers(version):
         Answer(answer_id="percentage-of-shopping", value=21, list_item_id="vhECeh")
         in data_payload["answers"]
     )
+
+
+def test_payload_supplementary_data():
+    questionnaire_store = get_questionnaire_store(AuthPayloadVersion.V2)
+
+    full_routing_path = [
+        RoutingPath(
+            ["dynamic-answer"],
+            section_id="section",
+        )
+    ]
+
+    supplementary_data = {
+        "survey_name": "supermarket test survey",
```

Comment:
Really minor but better to use one of the known supplementary data keys rather than survey_name?